### PR TITLE
feat: Get totals method

### PR DESCRIPTION
**Pull request by leonelmunozlopez**

### CODE REVIEW

**Comment by afiebig** [comment]
You have unresolved conflicts.

File: src/transbank.c [hunk 1]
```diff
@@ -107,6 +107,57 @@ BaseResponse *parse_load_keys_close_response(char *buf)
   return response;
 }
 
+TotalsResponse *parse_get_totals_response(char *buf)
+{
+  TotalsResponse *response = malloc(sizeof(TotalsResponse));
+
+  char *w;
+  int i = 1, l = 0, f = 0;
```
Comment (afiebig) [inline src/transbank.c]: Took me awhile to figure what was doing each of this variables, it would be a good idea to use more descriptive names, to make our life easier in the future.

File: src/transbank.c [hunk 2]
```diff
@@ -107,6 +107,57 @@ BaseResponse *parse_load_keys_close_response(char *buf)
...
+  for (int x = i; x < strlen(buf); x++)
+  {
+    if (buf[x] == '|')
+    {
+      w = malloc(l * sizeof(char *));
+      strncpy(w, buf + i, l);
+      w[l] = 0;
+
+      f++;
+      i = x + 1; // Set new position
```
Comment (afiebig) [inline src/transbank.c]: if you name `i` something like `word_start_position` you don't need this comment

File: src/transbank.h [hunk 3]
```diff
@@ -10,15 +10,17 @@
...
+enum TbkReturn
+{
     TBK_OK = 0,
     TBK_NOK = -1
 };
 
-extern enum TbkReturn open_port(char* portName, int baudrate);
-extern char* sale(int amount, int ticket, bool send_messages);
+extern enum TbkReturn open_port(char *portName, int baudrate);
+extern char *sale(int amount, int ticket, bool send_messages);
```
Comment (afiebig) [inline src/transbank.h]: This change is not related with the GetTotals, and should be separated, but first i think we should discuss if we need to change the pointer naming convention, since the rest of the code is using ìnt* i`

File: test/test_transbank.c [hunk 4]
```diff
@@ -4,111 +4,144 @@
...
-                        0x35, 0x39, 0x37, 0x30, 0x32, 0x39, 0x34, 0x31, 0x34, 0x33, 0x30, 0x30, 0x07,
-                        0x37, 0x35, 0x30, 0x30, 0x31, 0x30, 0x38, 0x37, 0x03, 0x30, '\0'};
-    strcpy(buf, response);
+int __wrap_read_bytes(struct sp_port *port, char *buf, Message message)
+{
+    char command[5];
+    strncpy(command, message.payload + 1, 4);
+
+    // Get totals
+    if (strcmp(command, "0700") == 0)
```
Comment (afiebig) [inline test/test_transbank.c]: This is a very good Solution 👏

File: src/transbank.h (hunk 3 above)
Comment (leonelmunozlopez) [reply src/transbank.h]: This was formatting
